docs/superpowers/plans/2026-04-25-architecture-stocktake.md

```
@@ -0,0 +1,1269 @@
+    D[1 · DATA INGEST<br/>Dukascopy + MT5 broker sync]
+    E[2 · EA DEFINITION<br/>schema + overrides]
+    S[3 · BACKTEST SWEEP<br/>random today; Pillar 2 adds Bayesian/CMA/walk-forward]
+    I[4 · INSPECT &amp; PICK<br/>Web UI + baseline compare]
+    DE[5 · DEPLOY TO VPS<br/>git push + scheduled task]
+    R[6 · RECONCILE LIVE ⇄ BACKTEST<br/>match/better/worse/missing/extra]
+
+    D --> E --> S --> I --> DE --> R
+    R -. drift &amp; alerts .-> I
+    DE -. MT5 data round-trip .-> D
+```
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)